stock_getter/backend_stockgetter/stock_service/tests.py

```diff
@@ -1,3 +1,71 @@
 from django.test import TestCase
+from django.urls import reverse, resolve
+from django.test import Client
+from .views import *
+
+
+class TestUrls(TestCase):
+    """Class for url tests.
```

Comment:
Can we minimize the use of comments? For this particular one we named class TestUrls so it is self explanatory that it is a "class for url tests".

In case below, if we believe that "This method checks the url to see if they are valid." we can name that test more self explanatory like test_check_if_valid_url.

Similar for other cases. Feel it would make the code a lot cleaner.

Reply:
I disagree. Comments are nice, and the standard practice in the industry is to comment as much as possible. That's actually encouraged. So I disagree with @SylvieXiang. We should include many comments.

Reply:
Can you point to the part of PEP8 that says this? I think it just says that if we do need to comment a block, it  should be done in that way, but if we don't need (and can avoid), it is not applicable. I read in a lot of places about the making code self explanatory and benefits of minimizing the comments unless absolutely necessary and I think it makes sense. Do you maybe have some link with reasons for writing a lot of comments.

If we focus on the first example, what additional information are we getting from the comment? Feel we can remove these at least.